Pacifique78/MyDiary
# #169642181 change routes which use check-Params middle-ware

#### What does this PR do?
This PR updates the API's routes

#### Description of Task to be completed
>- first check the params before checking other middle-wares

#### How should this be manually tested?
>- clone the repo
>- run `npm install` and `npm run dev` to start the server
>- use either postman or any other testing tool to see if all endpoints involving check-Params middleware are working:
>>>- ` http://localhost:4000/api/v2/entries/4` 
>>>- `http://localhost:4000/api/v2/entries/3`
>>>- `http://localhost:4000/api/v2/entries/4`
>- run `npm test` to see if the tests are passing
>- Check if the Travis test are passing

#### What are the relevant pivotal tracker stories?
#169642181 

#### Screenshots (if appropriate)
